experiment/coverage_utils.py

```diff
@@ -188,6 +189,14 @@ def get_coverage_infomation(coverage_summary_file):
         return json.loads(summary.readlines()[-1])
 
 
+def download_json_summary(experiment_name, dst_file):
+    """Downloads the json summary for |experiment_name| to |dst_file|."""
+    filestore_path = exp_utils.get_filestore_path()
+    src_file = posixpath.join(filestore_path, experiment_name, 'reports',
+                              'covered_regions.json')
+    filestore_utils.cp(src_file, dst_file)
+
+
```

Comment:
wondering if this function will be used by anything else than generate_report.py. if not, something like `analysis/coverage_data_utils.py` might be a better place for it.